spacy/gold.pyx

```diff
@@ -73,9 +73,15 @@ def merge_sents(sents):
     return [(m_deps, m_brackets)]
 
 
+_NORM_MAP = {"``": '"', "''": '"'}
```

Comment:
I'm not sure we want to be hard-coding the normalization here – I guess this should at least be an argument the user can change? Also, if we do normalization, do we want to handle this differently and actually substitute the characters? @honnibal

Reply:
I thought the same thing, but it turns out that this hard-coded in the existing _align code, so that's why it's implemented here. I think it would be a good opportunity to remove this since we're handling it more generally elsewhere now.

Reply:
Ahh I hadn't seen that (and it actually substitutes the characters, whereas this norm map change just substitutes the tokens?). But yes, in any case, I think we should definitely fix this then.

Reply:
It seems to substitutes the tokens in this line (https://github.com/explosion/spaCy/blob/118030444902f4696a5ace360d7854a3218dadb7/spacy/_align.pyx#L189)
If this normalization is unnecessary, I remove this.

Reply:
@adrianeboyd I think it's better to avoid mixing behavioural changes into refactors, because it makes it harder to go back and debug the refactor later. It's much easier if the refactor is supposed to give identical output.